libs/components/validation/src/lib/modules/url-validation/url-validation.directive.spec.ts

```diff
@@ -303,7 +309,9 @@ describe('URL validation via directive - non-onceability', () => {
     };
 
     const inputEl = element.querySelector('input');
-    inputEl.value = text;
+    if (inputEl) {
+      inputEl.value = text;
```

Comment:
Instead of putting `if` checks throughout the code, and the occasional `?`, we could create a new helper method that always returns the known input element.
```
function getInputElement(fixture: ComponentFixture<>): HTMLInputElement {
  return fixture.nativeElement.querySelector('input') as HTMLInputElement;
}
```